src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -26,6 +26,36 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NormalizingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | <xref:System.Single> or <xref:System.Double> |
+    /// | Output column data type | The same as the data type in the input column |
+    ///
+    /// The resulting NormalizingEstimator will normalize the data in one of the following ways based upon how it was created:
+    /// * Min Max - Linear rescale such that minimum and maximum values are mapped between -1 and 1.
+    /// * Mean Variance - Rescale to unit variance and, optionally, zero mean.
+    /// * Log Mean Variance - Rescale to unit variance on the log scale.
+    /// * Binning - Bucketize and then rescale to between -1 and 1.
```

Comment:
Is this correct? Isn't it between 0 and 1?

Reply:
it looks like -1 and 1. See the sample:
[https://github.com/dotnet/machinelearning/blob/ea5c095f90cced3bd2c67da7289fe1be532f9ba4/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeBinning.cs#L58](https://github.com/dotnet/machinelearning/blob/ea5c095f90cced3bd2c67da7289fe1be532f9ba4/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeBinning.cs#L58)


---
In reply to: [276894276](https://github.com/dotnet/machinelearning/pull/3432#discussion_r276894276) [](ancestors = 276894276)